CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Completions for `--class` and `-t` (short title)
 - Change the mouse cursor when hovering over the message bar and its close button
 - Support combined bold and italic text (with `font.bold_italic` to customize it)
+- Extra bindings for F13-F20
+- Bindings with combined modifiers
```

Comment:
Could this be more clear? I'm not sure which bindings this is referring too.

Reply:
I don't want this to be too bloated.

Reply:
I think it could be clarified a tiny bit without much extra wording.

Something like this maybe:

```
 - Terminal escape bindings with combined modifiers
```

That way people know just slightly more about it, but it's still short and clear.